srcpkgs/libunarr/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'libunarr'
+pkgname=libunarr
+version=1.0.1
+revision=1
+wrksrc="unarr-${version}"
```

Comment:
imo the package should be unarr as upstream calls it

Reply:
I'm not sure about this, since even the developer calls it libunarr in the readme, and if I wanted to install it from the repo, being a library, I would search for `lib`. But that's my opinion, if you want it changed tell me